sambaplayersdk/src/main/java/com/sambatech/player/utils/SharedPrefsUtils.java

```diff
@@ -0,0 +1,44 @@
+package com.sambatech.player.utils;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.text.TextUtils;
+
+final public class SharedPrefsUtils {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.sambatech.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)